internal/app/api/v1/events.go

```diff
@@ -8,7 +8,7 @@ import (
 	"github.com/gofiber/fiber/v2"
 	"github.com/gofiber/websocket/v2"
 
-	"github.com/fluxcd/pkg/runtime/events"
+	events "github.com/fluxcd/pkg/apis/event/v1beta1"
```

Comment:
Why did we change it?  Technically, this part is not used. We implemented this requirement using test triggers

Reply:
I updated the dependencies, and they moved this part of the code, so the api-server wasn't building anymore. I just updated the path, it's pointing to the same struct.